redirects-transition.conf

```diff
@@ -422,7 +429,12 @@ rewrite ^/law/litigation_CCA_X.shtml https://www.fec.gov/legal-resources/court-c
 
 # law/cfr/ redirects
 rewrite ^/law/cfr/cfr.shtml https://www.fec.gov/regulations redirect;
-rewrite ^/law/cfr/ej_citation.shtml https://www.fec.gov/legal-resources/regulations/explanations-and-justifications/explanations-and-justifications-citation-index/ redirect;
+rewrite ^/law/cfr/cfr_2008.pdf https://www.govinfo.gov/content/pkg/CFR-2008-title11-vol1/pdf/CFR-2008-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/cfr_2009.pdf https://www.govinfo.gov/content/pkg/CFR-2009-title11-vol1/pdf/CFR-2009-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/11_cfr.pdf https://www.govinfo.gov/content/pkg/CFR-2010-title11-vol1/pdf/CFR-2010-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/2014cfr.pdf https://www.govinfo.gov/content/pkg/CFR-2014-title11-vol1/pdf/CFR-2014-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/11cfr2015.pdf https://www.govinfo.gov/content/pkg/CFR-2015-title11-vol1/pdf/CFR-2015-title11-vol1.pdf redirect;
+xrewrite ^/law/cfr/ej_citation.shtml https://www.fec.gov/legal-resources/regulations/explanations-and-justifications/explanations-and-justifications-citation-index/ redirect;
```

Comment:
@kathycarothers Please change from `xrewrite` to `rewrite` at the beginning of this line.